src/otx/core/utils/tile_merge.py

```diff
@@ -22,6 +22,29 @@
 from otx.core.data.entity.instance_segmentation import InstanceSegBatchPredEntity, InstanceSegPredEntity
 from otx.core.data.entity.segmentation import SegBatchPredEntity, SegPredEntity
 
+MAX_ELEMENTS: int = 2**31 - 1
```

Comment:
Can be also done in cpp style to avoid explicit computations: `np.iinfo(np.int32).max`